collection-scripts/gather

```diff
@@ -39,6 +39,10 @@ resources+=(nodenetworkstates)
 # ImageStreamTag
 resources+=(istag)
 
+# Nodes and Machines
+resources+=(nodes)
```

Comment:
Nodes are expected to be collected by OCP must-gather base image:

https://github.com/openshift/must-gather/blob/master/collection-scripts/gather#L21

Reply:
is it expected to always run both images?

Reply:
I can't say it is always expected, since it is based on the user. It can be done by:

`oc adm must-gather --image=quay.io/kubevirt/must-gather --image=quay.io/openshift/origin-must-gather`

We used to skip resources that were collected by the base image, but can't say if that informal guidelines was maintained. 

I'd let someone of the active maintainer share his thought on it.

Reply:
Without these, the report is useless from an NMO perspective.  So I think we should include them.

Reply:
> I'd let someone of the active maintainer share his thought on it.

@masayag any news on this?

Reply:
If no one raised an objection for it - seems we can proceed with it.

Reply:
perfect, thanks